daemon/cluster/listen_addr.go

```diff
@@ -123,6 +123,22 @@ func validateDefaultAddrPool(defaultAddrPool []string, size uint32) error {
 	return nil
 }
 
+// validateDataPathPort validates vxlan udp port (data path port) number.
```

Comment:
Might want to add that;

- valid port numbers are between `1024` and `49151`
- if no port is set, the default (`4789`) is returned

Reply:
I already added those two comments inside the function.  Thought it will be redundant.
I can add it in API comment section too.